### PR TITLE
ICON AWS

### PR DESCRIPTION
**Pull request by johnny7861532**
#9 
Finished ICON on AWS

### CODE REVIEW

File: aws/ICON/README.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Setting up Wanchain on AWS
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: This should change to ICON

File: aws/ICON/README.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+> Wanchain aims to build a super financial market of digital assets. It is an infrastructure connecting different digital assets.
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: This should change to ICON description

File: aws/ICON/README.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    - vCPUs: 1
+    - Memory: 1 GB
+    - Storage: EBS only
+
+![Choose Instance](https://s3.amazonaws.com/kaizen-images/github/aws_choose_instance.png)
+
+Review the instance information
+![Review Instance](https://s3.amazonaws.com/kaizen-images/github/aws_review_instance.png)
+
+### Using AWS S3 to store Wanchain blockchain data
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: Change to ICON

File: aws/ICON/README.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+```
+sudo apt-get update
+```
+
+In order to install brew on Ubuntu we need to install some kit 
+```
+sudo apt-get install build-essential curl git ruby libbz2-dev \libcurl4-openssl-dev libexpat-dev libncurses-dev zlib1g-devtar xvfz gwan-linux-amd64-1.0.4-b7ce29ea.tar.gz
+```
+
+Install rabbitmq (Ubuntu )
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: `Install rabbitmq (Ubuntu)`

File: aws/ICON/README.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+```
+sudo apt-get install rabbitmq-server
+```
+
+Download install brew from source  to mount AWS S3
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: `Download install brew from source to mount AWS S3`

File: aws/ICON/README.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+[2018-12-07 09:33:26 +0000] [20729] [INFO] Booting worker with pid: 20729
+Started tbears service successfully
+
+```
+
+
+
+## Notice:
+
+The ndoe we trying to start is private chain, if you want to switch to another net you need to fix the uri section inside tbears_cli_config.json.
```
Comment (PhyrexTsai) [inline aws/ICON/README.md]: quote `tbears_cli_config.json`

**Review by PhyrexTsai — APPROVED** [review]

**Comment by johnny7861532** [comment]
@PhyrexTsai  finished fixing.

**Review by PhyrexTsai — APPROVED** [review]

**Pull request merged**